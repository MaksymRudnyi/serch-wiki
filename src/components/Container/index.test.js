const index = require("./index")
// @ponicode
describe("index.default", () => {
    test("0", () => {
        let callFunction = () => {
            index.default({ children: "Product Accountability Executive" })
        }
    
        expect(callFunction).not.toThrow()
    })

    test("1", () => {
        let callFunction = () => {
            index.default({ children: "Customer Metrics Consultant" })
        }
    
        expect(callFunction).not.toThrow()
    })

    test("2", () => {
        let callFunction = () => {
            index.default({ children: "National Infrastructure Supervisor" })
        }
    
        expect(callFunction).not.toThrow()
    })

    test("3", () => {
        let callFunction = () => {
            index.default({ children: "Principal Implementation Strategist" })
        }
    
        expect(callFunction).not.toThrow()
    })

    test("4", () => {
        let callFunction = () => {
            index.default({ children: "Senior Brand Assistant" })
        }
    
        expect(callFunction).not.toThrow()
    })

    test("5", () => {
        let callFunction = () => {
            index.default({ children: undefined })
        }
    
        expect(callFunction).not.toThrow()
    })
})
