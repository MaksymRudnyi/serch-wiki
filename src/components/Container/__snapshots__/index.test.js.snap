// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component without props should match snapshot 1`] = `
<Container>
  <Children
    articles={Array []}
    onSearchChange={[MockFunction]}
    searchValue="test"
  >
    <div>
      Test
    </div>
  </Children>
</Container>
`;
